src/commands/create/index.ts

```diff
@@ -0,0 +1,259 @@
+        availableLanguages = packageIndex.getAvailableLanguages();
+        availableExamples = packageIndex.getAvailableExamples();
+        availableInterfaces = packageIndex.getAvailableInterfaces();
+        Create.flags.language.options = availableLanguages.map((languageEntry: string) => {
+            return languageEntry;
+        });
+        Create.flags.interface.options = availableInterfaces.map((interfaceEntry: FunctionalInterfaceDescription) => {
+            return interfaceEntry.value;
+        });
+    }
+
+    private async _setDefaultAppManifestInterfaceConfig(interfaces: string[]) {
+        if (this.appManifestInterfaces.interfaces.length > 0) {
+            return;
+        }
+        const interfacesToUse =
+            Array.isArray(interfaces) && interfaces.length
+                ? availableInterfaces.filter((interfaceEntry) => interfaces.includes(interfaceEntry.value))
+                : availableInterfaces;
+
+        for (const interfaceEntry of interfacesToUse) {
+            const defaultAppManifestInterfaceConfig: AppManifestInterfaceEntry = {
+                type: interfaceEntry.value,
+                config: {},
+            };
+
+            for (const arg of interfaceEntry.args) {
+                defaultAppManifestInterfaceConfig.config[arg.id] =
+                    arg.type === 'object' && arg.default ? JSON.parse(arg.default) : arg.default;
+            }
+
+            this.appManifestInterfaces.interfaces.push(defaultAppManifestInterfaceConfig);
+        }
+    }
+
+    private _getScriptExecutionPath(sdkConfig: SdkConfig): string {
+        const basePath = process.env.VELOCITAS_SDK_PATH_OVERRIDE
+            ? process.env.VELOCITAS_SDK_PATH_OVERRIDE
+            : join(sdkConfig.getPackageDirectory(), 'latest');
+
+        return join(basePath, '.project-creation', 'run.py');
+    }
+
+    async run(): Promise<void> {
+        const packageIndex = PackageIndex.read();
+        this._loadDataFromPackageIndex(packageIndex);
+        const { flags } = await this.parse(Create);
+        this.log(`Creating a new Velocitas project ...`);
+
+        if (Object.keys(flags).length === 0) {
+            this.log('Interactive project creation started');
+            await this._runInteractiveMode(flags);
+        }
+
+        if (!flags.name) {
+            throw new Error("Missing required flag 'name'");
+        }
+        if (!flags.language) {
+            throw new Error("Missing required flag 'language'");
+        }
+
+        if (!flags.example) {
+            this._setDefaultAppManifestInterfaceConfig(flags.interface!);
+        }
+
+        await ProjectConfig.create(packageIndex.getExtensions(), flags.language, this.config.version);
+        await createAppManifest(flags.name, this.appManifestInterfaces);
+        const sdkConfig = new SdkConfig(flags.language);
+        await sdkDownloader(sdkConfig).downloadPackage({ checkVersionOnly: false });
+
+        await awaitSpawn(
```

Comment:
We need to check the exit code and exit create if the create script is not available / returns non 0 as exit code.